README.md

```diff
@@ -32,10 +32,14 @@ Download and install Git
  Use **PowerSell** in Administrator Mode
  
  > go through the **Requirements** found  [here](https://chocolatey.org/install) Before installing **Meteor**
```

Comment:
I think it would be awesome to have just the first letter as uppercase in case of those who mind much on typography 😉

> Go through the **requirements** found  [here](https://chocolatey.org/install) before installing **meteor**